.github/workflows/cal-importer.yml

```diff
@@ -39,18 +39,3 @@ jobs:
         run: aws s3 sync ./cal/ s3://ledger-live-cryptoassets-prd/cryptoassets --delete --acl public-read
       - name: Cache invalidate PRD
         run: aws cloudfront create-invalidation --distribution-id E53IORS7MW883 --paths "/cryptoassets/*" --debug
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          role-to-assume: ${{ secrets.AWS_ASSETS_RUNNER_ROLE_STG }}
-          aws-region: eu-west-1
-          role-duration-seconds: 3600
```

Comment:
Just for my understanding, why don't we need the same before task `Deploy json to s3 PRD`?

Reply:
It's already config in runner according to infra